com.playeveryware.eos/Documentation~/google_signin.md

```diff
@@ -0,0 +1,54 @@
+<a href="/com.playeveryware.eos/README.md"><img src="/com.playeveryware.eos/Documentation~/images/PlayEveryWareLogo.gif" alt="README.md" width="5%"/></a>
+
+# <div align="center">Connect Login with GoogleID using Credential Manager</div>
+---
+
+## Overview
+
+This document describes how to enable Google ID authentication in conjunction with Epic Online Services Plugin for Unity.  
```

Comment:
Given that this article is about "Connect Login", would it be worthwhile to use specific language about the Connect interface rather than "authentication"?